apps/src/lib/levelbuilder/course-editor/ResourcesEditor.js

```diff
@@ -106,6 +106,7 @@ export default class ResourcesEditor extends Component {
           <div style={{marginBottom: 5}}>Preview:</div>
           <TeacherResourcesDropdown
             resources={resources.filter(x => !!x.type)}
+            useMigratedResources={false}
```

Comment:
Why is this? it makes me think that the course editor will always use non-migrated resources, but earlier i see the course editor checking props.useMigratedResources.

Reply:
This component is only used for old migrated resources -- it's the one that currently shows up on the course edit and unit edit pages. This whole file has an expiration date on it and won't ever use migrated resources.

Would it be worth it for me to move the "new" `ResourcesEditor`? Currently it lives in `lesson-editor/` but that increasingly seems like not the right place for it. I could move it as a follow-up to this PR

Reply:
> This component is only used for old migrated resources

do you mean old, _un_-migrated resources? sorry to pick on a likely typo, but just want to be sure there's not a dimension to this that I'm missing.

> Would it be worth it for me to move the "new" ResourcesEditor? Currently it lives in lesson-editor/ but that increasingly seems like not the right place for it.

I think the hardest part for me in reading this code is figuring out what's old / unmigrated and what's new / migrated. Generally yes, some kind of distinction would be great, so that it's clear from the file name + path whether it is old or new.  If this file has an expiration date on it, then renaming the file with a prefix of Unmigrated, Legacy or Deprecated could be another option. I'm only suggesting that because I'm not quite sure what's on the table for what directory you'd move this to -- however moving it could be fine too if you have a vision for that.

Reply:
looking at the rest of the PR again, I think I'd prefer the filename / component name to make the distinction, so that it's clear from wherever the component is instantiated that it is legacy / deprecated.

Reply:
just one more thought to add here -- if we are consistent about naming the new components Migrated, then it seems ok not to rename the old components. :-)

Reply:
> So in this case, rename this file/component to be LegacyResourcesEditor?

That's what I was suggesting, however it looks like the name scheme you are generally using is that no prefix means legacy and "migrated" prefix means migrated, so if you are consistent about that them I'm ok with it.